.pre-commit-config.yaml

```diff
@@ -54,5 +55,5 @@ repos:
           - --max-line-length=120
           - --min-public-methods=0
           - --good-names=o,w,q,f,fp,i,e
-          - --disable=E0401,W1201,W1203,C0114,C0115,C0116,C0411,W0107,W0511,W0702,R0801,R1705,R1710
+          - --disable=E0401,W1201,W1203,C0114,C0115,C0116,C0411,W0107,W0511,W0702,R0801,R1705,R1710,W0201
```

Comment:
adding this for now, as the issue occurs again. This will need to be more investigated separately.